app/views/components/forms/question_types/_radio_buttons.html.erb

```diff
@@ -9,6 +9,10 @@
       <%= label_tag(nil, for: "#{question.answer_field}_#{(index + 1).to_s}", class: "usa-radio__label") do %>
         <%= option.text %>
       <% end %>
+      <% if ["OTHER", "OTRO"].include?(option.text.upcase) %>
```

Comment:
this works if the 'other' radio is selected, then the value will be 'other', even if there is text in the text field.

the value of the response should be the typed in value, or if blank, then the value of the `other` text

Reply:
and, got specs? 

🔴  🟢

Reply:
Confirmed radio button works with specs